code/game/objects/items/stacks/rods.dm

```diff
@@ -47,7 +47,6 @@ var/global/list/datum/stack_recipe/rod_recipes = list ( \
 
 		if(WT.remove_fuel(0,user))
 			var/obj/item/stack/sheet/metal/new_item = new(user.loc)
-			new_item.add_to_stacks(user)

```

Comment:
Can you explain why this is removed?

Reply:
When a new stack's initialized / dropped on the floor / put inside a container, they stack together automatically with the PR. The previous function basically did this manually.